schema/artwork/index.js

```diff
@@ -173,6 +173,30 @@ const ArtworkType = new GraphQLObjectType({
         description: 'Whether work can be purchased through e-commerce',
         resolve: ({ acquireable }) => acquireable,
       },
+      is_buy_nowable: {
+        type: GraphQLBoolean,
+        description: 'When in an auction, can the work buy bought before any bids are placed',
+        resolve: ({ id, acquireable }) => {
+          return gravity(`related/sales`, { size: 1, active: true, artwork: [id] })
+            .then(_.first)
+            .then(sale => {
+              if (!sale) return [false];
```

Comment:
Admittedly somewhat confusing: The next then destructures an iterable, so this is essentially the value for `sale`. It's a lil weird to break out of a Promise chain and this seemed preferable to me. I suppose I could also return `[sale]` here which might be a bit more clear.